sumcheck/src/prover.rs

```diff
@@ -1,13 +1,12 @@
-use std::{array, mem, sync::Arc};
+use std::{mem, sync::Arc};
 
 use ark_std::{end_timer, start_timer};
 use crossbeam_channel::bounded;
 use ff_ext::ExtensionField;
 use itertools::Itertools;
 use multilinear_extensions::{
-    commutative_op_mle_pair,
-    mle::{DenseMultilinearExtension, MultilinearExtension},
-    op_mle, op_mle_product_3, op_mle3_range,
+    mle::{DenseMultilinearExtension, FieldType, MultilinearExtension},
```

Comment:
(not in this line)
we can recover many ignored tasks `#[ignore = "temporarily not supporting degree > 2"]` in this file
https://github.com/scroll-tech/ceno/blob/master/sumcheck/src/test.rs#L104